docs/topics/js/js-ir-compiler.md

```diff
@@ -72,6 +71,37 @@ kotlin {
 }
 ```
 
+## Lazy initialization of top-level properties
+
+> Lazy initialization of top-level properties is [Experimental](components-stability.md). It may be dropped or changed at any time.
+> Opt-in is required (see the details below), and you should use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-44320).
+>
+{type="warning"}
+
+For better application startup performance, the Kotlin/JS IR compiler offers an option to initialize top-level properties
+lazily. This way, the application loads without initializing all the top-level properties used in its code. It initializes
+only the ones needed at startup; other properties get initialized later when the code that uses them is executed. 
```

Comment:
The second part of the sentence is too heavy. Could we get rid of this passive voice?

Sorry, can't find a better solution :/